common/OpTestHost.py

```diff
@@ -1180,6 +1180,110 @@ def host_build_fwts_tool(self, i_dir):
             raise OpTestError(l_msg)
 
     ##
+    # @brief Check that host has a CAPI FPGA card
+    #
+    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
+    #
+    def host_has_capi_fpga_card(self):
+        l_cmd = "lspci -d \"1014:0477\""
+        print l_cmd
+        l_res = self.host_run_command(l_cmd)
+        print l_res
+        if (l_res.__contains__('IBM Device 0477')):
+            l_msg = "Host has a CAPI FPGA card"
+            print l_msg
+        else:
+            l_msg = "Host has no CAPI FPGA card"
+            print l_msg
+            raise OpTestError(l_msg)
+
+    ##
+    # @brief Clone latest cxl-tests git repository in i_dir directory
+    #
+    # @param i_dir @type string: directory where cxl-tests will be cloned
+    #
+    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
+    #
+    def host_clone_cxl_tests(self, i_dir):
+        l_msg = "https://github.com/ibm-capi/cxl-tests.git"
```

Comment:
Looks like ibm-capi/cxl-tests repository is not open to all, it is asking for username/password. cloning is failing. Then it will be difficult to run this tests for all the users.